docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Metrics"
+linkTitle: "Metrics"
+weight: 03
+type: "docs"
+---
+
+The metric configuration defines which metric type is watched by the Autoscaler.
+
+## Setting metrics per revision
+
+For [per-revision](./autoscaling-concepts.md) configuration, this is determined using the `autoscaling.knative.dev/metric` annotation.
+The possible metric types that can be configured per revision depend on the type of Autoscaler implementation you are using:
+
+* The default KPA Autoscaler supports the `concurrency` and `rps` metrics.
+* The HPA Autoscaler supports the `concurrency`, `rps` and `cpu` metrics.
```

Comment:
Can you describe what these options mean?

Reply:
In particular, how concurrency and RPS differ.

Feel free to defer with a `<!-- TODO: ... -->` if you don't have bandwidth right now.

Reply:
Added a TODO note :) I think I had pretty much the same note previously and removed it because I knew I wouldn't get to it :joy:

Reply:
A TODO lets someone else pick up the item if you don't get back to it. 😁